ospfd/ospf_te.c

```diff
@@ -1551,6 +1554,10 @@ ospf_mpls_te_lsa_refresh (struct ospf_lsa *lsa)
   else
     top = ospf_lookup ();
 
+  if (OSPF_DEBUG_TRACE)
```

Comment:
The purpose of the log is for daemon internal activity. Started using as part of vrf development. If any objection, I will remove it.